code/modules/clothing/spacesuits/rig/suits/skrell.dm

```diff
@@ -51,13 +53,13 @@
 	light_overlay = "tup2_rig_helmetlight"
 
 /obj/item/rig/skrell/tup
-	name = "outdated tupkala suit control module"
-	desc = "A suit control module designed by the Nralakk Federation for Tup operations. The best of the best, albeit outdated now."
+	name = "tupkala infiltration suit control module"
+	desc = "An old yet reliable suit control module, designed by the Nralakk Federation for clandestine operations. The best of the best, albeit outdated now."
 	icon = 'icons/clothing/rig/tup.dmi'
 	icon_state = "tup_rig"
 	suit_type = "tup suit"
 	armor = list(
-		melee = ARMOR_MELEE_VERY_HIGH,
+		melee = ARMOR_MELEE_MAJOR,
 		bullet = ARMOR_BALLISTIC_RIFLE,
 		laser = ARMOR_LASER_RIFLE,
```

Comment:
This is some ridiculous armour for a ninja hardsuit.

Reply:
There are other ninja rigs with similar or better armor values, and lore-wise it makes sense that the Tups would have high-quality shit. I am not an expert on armor balance however, so I can tone it down if need be.

Reply:
ninja version has been nerfed to be more in line with normal combat hardsuit armor, good armor is saved for the funny inaccessible one